functions/Get-DbaAgentJob.ps1

```diff
@@ -47,9 +47,9 @@ FUNCTION Get-DbaAgentJob {
 			Returns all SQl Agent Job for the local and sql2016 SQL Server instances
 
 		.EXAMPLE
-			Get-DbaAgentJob -SqlInstance localhost -Enabled True
+			Get-DbaAgentJob -SqlInstance localhost -IncludeDisabled
 
```

Comment:
This needs to be separate `.EXAMPLE`. Look at some of the other Get commands will show you the common examples we include. It is best to try and do an example for each parameter. We obviously forgot to show examples for the Job and ExcludeJob parameter so if you want to add those as well would be good.